zip-0227.rst

```diff
@@ -1,8 +1,359 @@
+  
+================= ================== ========================== ========================================================================
+Size              Name               Data Type                  Description
+================= ================== ========================== ========================================================================
+Varies            asset_desc         byte                       Uni-code encoded string of varied size, up to 512 bytes
+Varies            nNotes             compactSize                The number of notes in the issuance action
+noteSize * nNotes vNotes             Note[nNotes]               A sequence of note descriptions within the issuance action
+1 bit             finalize           boolean                    The boolean that determines the finality of the issuance of that asset
+================= ================== ========================== ========================================================================
+
+We note that the output note commitment of the recipient's notes are not included in the actual transaction, but when added to the global state of the chain, they will be added to the `NoteCommitmentTree` as a shielded note. This prevents future usage of the note from being linked to the issuance transaction, as the nullifier key is not known to the validators and chain observers.
+
+Issuance Bundle
+---------------
+
+An issuance bundle, `IssueBundle`, is the aggregate of all the issuance-related information. Specifically, contains all the issuance actions and the issuer signature on the transaction SIGHASH that validates the issuance itself. It contains the following fields:
+
+- `ik`: the `issuanceValidatingKey`, that allows the validators to verify that the `AssetId` is properly associated with the issuer
+- `actions`: an array of issuance actions, of type `IssueAction`
+- `authorization`: the signature of the transaction SIGHASH, signed by the `issuanceAuthorizingKey`, `isk`, that validates the issuance 
+
+The issuance bundle is then added within the transaction format as a new bundle. This structure will be added to the v6 transaction format.
+
+======= ================== ========================== =========================================================================
+Bytes              Name               Data Type                  Description
+======= ================== ========================== =========================================================================
+Varies  nIssueActions      compactSize                The number of issuance actions in the bundle
+Varies  vIssueActions      IssueAction[nIssueActions] A sequence of issuance actions descriptions
+32      ik                 byte[32]                   The public verification key of the issuer, used to validate the signature
+64      authorization      byte[64]                   The signature of the transaction SIGHASH, signed by the issuer
```

Comment:
Each `IssueAction` can have a different asset description, which corresponds to a different `ik` and `authorization`.

So I think `ik` and `authorization` need to move to `IssueAction`.

Reply:
So that meets this requirement:
> - The asset identification should be unique and different issuer keys should not be able to generate the same asset identifier

But we also need to add a requirement that the same issuer key can be used to issue different assets in the same transaction (or issuance bundle in this case).

Reply:
Added this